x-pack/plugins/secops/public/components/page/hosts/events_table/index.tsx

```diff
@@ -80,37 +144,43 @@ const getEventsColumns = (startDate: number, endDate: number) => [
     sortable: true,
     truncateText: true,
     hideForMobile: true,
-    render: (item: EventItem) => (
-      <ProviderContainer>{getOr('--', 'event.type', item)}</ProviderContainer>
-    ),
+    render: ({ event }: { event: Ecs }) => <>{getOr('--', 'event.type', event)}</>,
```

Comment:
There is a subtle but funny little trick with `fp.getOr()` where if a value of a field happens to be `null` and not `undefined` it actually returns the null value back to the dom. So for example this can and will return a `null` when graphQL tells it event type to be null and then you won't get the `--` under event type.

You can see the weird blank values in my screen shot below:
<img width="1309" alt="screen shot 2019-01-09 at 3 46 37 pm" src="https://user-images.githubusercontent.com/1151048/50935140-4c212a00-1428-11e9-8454-ce2e88313b8a.png">

Since we are going to continuously run into this bug and the verbosity of `getOr` kind of not being exactly what we want, I suggest we write a ticket where one of us just write a new `getOr` for rendering in utils that works with `null` and `undefined` and just takes two arguments like so:

```ts
getOrEmpty('event.type', event)
```
And then we don't have all this messy `getOr` + `defaultTo` that we've been using.

Reply:
I belive that formatSafely should resolve it

Reply:
Your getOrEmpty was giving me some weird type error, but I agree we should resolve it for once at the sametime